roborock/protocols/b01_protocol.py

```diff
@@ -23,7 +23,7 @@
 
 def encode_mqtt_payload(dps: int, command: CommandType, params: ParamsType) -> RoborockMessage:
     """Encode payload for B01 commands over MQTT."""
-    dps_data = {"dps": {dps: {"method": command, "params": params or []}}}
+    dps_data = {"dps": {dps: {"method": str(command), "msgId": "1751755654575", "params": params or []}}}
```

Comment:
can you extract this msg id to a constant and give it a self documenting name that explains what it is?